healsparse/healSparseMap.py

```diff
@@ -1300,18 +1313,20 @@ def _degrade(self, nside_out, reduction='mean', weights=None):
 
     def degrade(self, nside_out, reduction='mean', weights=None):
```

Comment:
Do we want to change the name of the function to `ud_grade`?

Reply:
I guess we want to keep `degrade` for legacy purposes, but maybe also create a `ud_grade` to follow healpy's convention?

Reply:
At first I had a separate `upgrade` method, and I could easily be persuaded to use that directly.  I really dislike the `ud_grade` name in healpy, so I personally would vote against that.

Reply:
upgrade and degrade are now separated.